charts/renku/requirements.yaml

```
@@ -2,27 +2,27 @@ dependencies:
 - name: renku-core
   alias: core
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
-  version: 0.12.1
+  version: 0.12.2
 - name: renku-ui
   alias: ui
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
-  version: 0.11.2
+  version: 0.11.3
 - name: renku-notebooks
   alias: notebooks
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
-  version: 0.8.5
+  version: 0.8.5-c35d522
```

```suggestion
  version: 0.8.6
```
`0.8.6` has been fixed now.